src/components/item/settings/DownloadChatButton.js

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from 'react';
+import { PropTypes } from 'prop-types';
+import { CSVLink as CsvLink } from 'react-csv';
+import { useTranslation } from 'react-i18next';
+
+import { IconButton, Tooltip } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import { GetApp } from '@material-ui/icons';
+
+import { Button } from '@graasp/ui';
+import { normalizeMentions } from '@graasp/chatbox';
+
+import { hooks } from '../../../config/queryClient';
+
+import { EXPORT_CSV_HEADERS } from '../../../config/constants';
+import { DOWNLOAD_CHAT_BUTTON_ID } from '../../../config/selectors';
+
+import { BUTTON_VARIANTS } from '../../../enums';
+
+const useStyles = makeStyles({
+  link: {
+    textDecoration: 'none',
+    width: 'fit-content',
+  },
+});
+
+// todo: convert this into a backend call
+const DownloadChatButton = ({ chatId, variant }) => {
+  const [filename, setFilename] = useState('');
+  const { t } = useTranslation();
+  const classes = useStyles();
+
+  const { data: exportedChat, isLoading } = hooks.useExportItemChat(chatId);
```

Comment:
The problem using a hook is that you would always export the chat even tho the user didn't click on the button.